packages/circuit-ui/components/ImageInput/ImageInput.mdx

```diff
@@ -17,8 +17,34 @@ The ImageInput component allows users to upload images. It can be used on its ow
 
 Refer to the `Stateful` component story for a simplified code usage example.
 
-- **Do** use the `onChange` callback to immediately trigger an image upload when the user has selected one, even when the component is being used as part of a form. Faster form submissions add to a delightful user experience.
-- **Do not** use the ImageInput component if the image is read-only for the user. Instead, use the visual component from the ImageInput's `component` prop by itself, for example the [Avatar](Components/Avatar).
```

Comment:
`readonly` is a valid form input state in some cases (indicates to the user that there is a form element, only it can't be edited—should usually be accompanied with reasoning e.g. "View only, <a href="#">request edit access</a>"), so we probably shouldn't be recommending this